.build-harness

```diff
@@ -122,12 +122,13 @@ build-harness/shell builder pr/pre-commit pr/readme pr/github-update: RUNNER_DOC
 build-harness/shell builder: build-harness/runner
```

Comment:
Do not commit this file

Reply:
22b0ad5a1ad790df7362a8fba8175376764e24fc